extension/src/experiments/columns/like.ts

```diff
@@ -2,25 +2,35 @@ import { Column } from '../webview/contract'
 
 export type ColumnLike = {
   firstValueType?: string
+  description?: string
   label: string
   path: string
 }
 
-const starredColumnLike: ColumnLike = {
+export const starredColumnLike: ColumnLike = {
+  description: '$(star-full)',
   firstValueType: 'boolean',
-  label: '$(star-full)',
+  label: 'Starred',
   path: 'starred'
 }
 
-export const addStarredToColumns = (
-  columns: Column[] | undefined
+export const tagsColumnLike: ColumnLike = {
+  description: '$(git-commit)',
+  firstValueType: 'tags',
+  label: 'Tag',
```

Comment:
Should we say `Git Tag` or would it be obvious that the tags are related to git?